.github/workflows/ci-core-reusable.yml

```diff
@@ -99,13 +99,16 @@ jobs:
           ci_run git config --global --add safe.directory /usr/src/zksync/contracts
 
           ci_run zk
-          ci_run zk init
+          ci_run zk init --chain-id-hack
 
-      # `sleep 30` because we need to wait until server added all the tokens
+      # `sleep 60` because we need to wait until server added all the tokens
       - name: Run server
         run: |
-          ci_run zk server --uring --components api,tree,eth,state_keeper,housekeeper &>server.log &
-          ci_run sleep 30
+          ci_run zk server --uring --components api,tree,eth,state_keeper,housekeeper,commitment_generator &>server.log &
+          ci_run sleep 60
```

Comment:
let's add a linear ticket to replace it with something that actively queries the server (and add a TODO here).
(otherwise, we risk that this test might become flaky over time).

Reply:
I'll create a task for the Platform team https://linear.app/matterlabs/issue/PLA-886/ensure-that-the-loadtest-waits-for-the-server-to-start